sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -380,4 +380,13 @@ export class EducationProgram extends RecordDataModel {
     referencedColumnName: "id",
   })
   submittedBy: User;
+
+  /**
+   * Field of study code for Education Program.
+   */
+  @Column({
+    name: "field_of_study_code",
+    nullable: true,
+  })
+  fieldOfStudyCode?: number;
```

Comment:
Please referer to the SQL migration file comment (https://github.com/bcgov/SIMS/pull/1362/files#r943789606). Is this field optional?

Reply:
I will update.

Reply:
Updated.